packages/api/src/metrics.js

```diff
@@ -0,0 +1,74 @@
+import { gql } from '@web3-storage/db'
+
+/**
+ * Retrieve metrics in prometheus exposition format.
+ * https://prometheus.io/docs/instrumenting/exposition_formats/
+ * @returns {Promise<string>}
+ */
+export async function metricsGet (request, env) {
+  const { findMetrics } = await env.db.query(gql`
+    query findMetrics {
+      findMetrics {
+        usersTotal,
+        uploadsTotal,
+        contentTotalBytes,
+        dealsActiveTotal,
+        dealsQueuedTotal,
+        pinsTotal,
+        pinsTotalBytes,
+        pinsQueuedTotal,
+        pinsPinningTotal,
+        pinsPinnedTotal,
+        pinsFailedTotal
+      }
+    }
+    `)
```

Comment:
As the dataset gets bigger we might need to split this out into multiple queries.

Reply:
so probably split between:

- `findUserMetrics`
  - `usersTotal`
- `findUploadMetrics`
  - `uploadsTotal`
  - `contentTotal`
  - `contentTotalBytes`
- `findPinMetrics`
  - `pinsTotal`
  - `pinsTotalBytes`
  - `pinsQueuedTotal`
  - `pinsPinningTotal`
  - `pinsPinnedTotal`
  - `pinsFailedTotal`
- `findDealMetrics`
  - `dealsActiveTotal`
  - `dealsQueuedTotal`

wdyt?